docs/paper/paper.md

```diff
@@ -0,0 +1,104 @@
+---
+title: TorchMetrics - Measuring Reproducibility in PyTorch
+tags:
+  - python
+  - deep learning
+  - pytorch
+authors:
+  - name: Nicki Skafte Detlefsen
+    affiliation: 1,2
+  - name: Jiri Borovec
+    affiliation: '1'
+  - name: Justus Schock
+    affiliation: 1,3
+    orcid: 0000-0003-0512-3053
+  - name: Ananya Harsh Jha
+    affiliation: '1'
+  - name: Teddy Koker
+    affiliation: '1'
+  - name: Luca Di Liello
+    affiliation: '4'
+  - name: Daniel Stancl
+    affiliation: '5'
+  - name: Changsheng Quan
+    affiliation: '6'
+  - name: William Falcon
+    affiliation: '1,7'
```

Comment:
In the case of multiple affiliations, should it be unified? I.e. either '1,2' or 1,2? (See lines 9, 13 and 26)

Reply:
They need to be with quotes as stated by the template :)

Reply:
fixed :)